jenkins-agents/jenkins-agent-mvn/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM quay.io/openshift/origin-jenkins-agent-maven:4.9
 
 ADD settings.xml $HOME/.m2/settings.xml
+
+USER root
```

Comment:
@palla714 Refering this Dockerfile, can we change this line to `USER 1001` and see if it resolves the permission denied issue ?

https://github.com/redhat-cop/containers-quickstarts/blob/master/jenkins-agents/jenkins-agent-python/Dockerfile#L29

Reply:
I have changed the user to 1001 and tested running it on local. It is creating image.